src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,47 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.dto.response.ShoppingCartResponseDto;
+import mate.academy.spring.service.MovieSessionService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/shopping-carts")
+public class ShoppingCartController {
+    private final MovieSessionService movieSessionService;
+    private final ShoppingCartService shoppingCartService;
+    private final UserService userService;
+    private final DtoResponseMapper<ShoppingCartResponseDto,
+            ShoppingCart> shoppingCartDtoResponseMapper;
+
+    public ShoppingCartController(MovieSessionService movieSessionService,
+            ShoppingCartService shoppingCartService, UserService userService,
+            DtoResponseMapper<ShoppingCartResponseDto,
+                    ShoppingCart> shoppingCartDtoResponseMapper) {
+        this.movieSessionService = movieSessionService;
+        this.shoppingCartService = shoppingCartService;
+        this.userService = userService;
+        this.shoppingCartDtoResponseMapper = shoppingCartDtoResponseMapper;
+    }
+
+    @PutMapping("/movie-sessions")
+    public String add(@RequestParam Long userId,
+            @RequestParam Long movieSessionId) {
```

Comment:
```suggestion
    public String add(@RequestParam Long userId,
                                @RequestParam Long movieSessionId) {
```

We do not add shopingCart, that is, an update would be more logical here, or addMovieSession